modules/ipi-install-preparing-the-bare-metal-node.adoc

```diff
@@ -53,75 +58,68 @@ $ vim bmh.yaml
-       address:
-       - ip: 192.168.123.16
-         prefix-length: 24
-       enabled: true
-       dhcp: false
+  routes:
+   config:
+   - destination: 0.0.0.0/0
+     next-hop-address: 192.168.123.1
+     next-hop-interface: enp0s4
+  dns-resolver:
+   config:
+    server:
+    - 192.168.123.1
+  interfaces:
+  - name: enp0s4
+    state: up
+    ipv4:
+     address:
+     - ip: 192.168.123.10
+       prefix-length: 24
+     enabled: true
+     dhcp: false
 ---
 apiVersion: v1
 kind: Secret
 metadata:
-  name: openshift-worker-<num>-bmc-secret <2> 
+  name: openshift-worker-<num>-bmc-secret
   namespace: openshift-machine-api
 type: Opaque
 data:
-  username: <base64-of-uid> <3>
-  password: <base64-of-pwd> <4>
+  username: <base64_of_uid>
+  password: <base64_of_pwd>
 ---
 apiVersion: metal3.io/v1alpha1
 kind: BareMetalHost
 metadata:
-  name: openshift-worker-<num> <2>
+  name: openshift-worker-<num>
   namespace: openshift-machine-api
 spec:
-  online: true
-  bootMACAddress: <NIC1-mac-address> <5>
+  online: True
+  bootMACAddress: <NIC1_mac_address>
   bmc:
-    address: <protocol>://<bmc-ip> <6>
-    credentialsName: openshift-worker-<num>-bmc-secret <2>
-    disableCertificateVerification: true <7>
-    username: <bmc-username> <8>
-    password: <bmc-password> <9>
-  preprovisioningNetworkDataName: openshift-worker-<num>-network-config-secret <10> <2>
-----
-<1> Optional: To configure network for a newly created node, specify the name of the secret that contains the network configuration. Follow the `nmstate` syntax to define the network configuration for your node.
-<2> Replace `<num>` for the worker number of the bare metal node in the `name` fields, the `credentialsName` field, and the `preprovisioningNetworkDataName` field.
-<3> Replace `<base64-of-uid>` with the `base64` string of the user name.
-<4> Replace `<base64-of-pwd>` with the `base64` string of the password.
-<5> Replace `<NIC1-mac-address>` with the MAC address of the bare metal node's first NIC. See the "BMC addressing" section for additional BMC configuration options.
-<6> Replace `<protocol>` with the BMC protocol, such as IPMI, RedFish, or others. Replace `<bmc-ip>` with the IP address of the bare metal node's baseboard management controller.
-<7> To skip certificate validation, set `disableCertificateVerification` to `true`.
-<8> Replace `<bmc-username>` with the `bmc` string of the user name.
-<9> Replace `<bmc-password>` with the `bmc` string of the password.
-<10> Optional: Provide the network configuration secret name in the `preprovisioningNetworkDataName` of the `BareMetalHost` CR if you have configured network for the newly created node.
+    address: <protocol>://<bmc_ip>
+    credentialsName: openshift-worker-<num>-bmc-secret
+    disableCertificateVerification: True
+    username: <bmc_username>
+    password: <bmc_password>
+  rootDeviceHints:
+    deviceName: <root_device_hint>
+  preprovisioningNetworkDataName: openshift-worker-<num>-network
```

Comment:
This must reference the first secret in the file, which in this example is called: openshift-worker-<num>-network-config-secret